src/App.js

```diff
@@ -35,6 +35,7 @@ function App() {
           <Route path="/validation" component={ValidationAccount} />
           <Route path="/statistiques-nationales" component={StatistiquesNationales} />
           <Route path="/statistiques/:type/:id/:dateDebut/:dateFin" component={EnregistrerStatistiquesPdf} />
+          <Route path="/statistiques/:type/:dateDebut/:dateFin" component={EnregistrerStatistiquesPdf} />
```

Comment:
@MorganP-projects Pour les paramètres optionnels , il y a les ( ) normalement. Un truc du genre <Route path="to/page(/:pathParam)" component={MyPage} />

Reply:
sinon ne pas hésiter à changer la racine. 

`<Route path="/statistiques-nationales/dateDebut/:dateFin" component={EnregistrerStatistiquesPdf} />`